src/main/java/com/marek/onlinebookstore/service/user/UserServiceImpl.java

```diff
@@ -14,17 +17,21 @@
 @Service
 @RequiredArgsConstructor
 public class UserServiceImpl implements UserService {
+    private static final RoleName USER = RoleName.ROLE_USER;
```

Comment:
It's better to replace `USER` constant  with `RoleName.ROLE_USER` as it refers to the actual enumeration.